portal/src/app/components/protocol-query/ProtocolQuery.tsx

```diff
@@ -131,17 +145,46 @@ export const ProtocolQuery: React.FC<ProtocolQueryProps> = (props: ProtocolQuery
                 required
                 customComponent={{
                   Option: (props: any) =>
-                    <IterationsSelectorCustomOption
+                    <SelectorCustomOption
                       {...props}
-                      showInputOption={showInputOption}
-                      setShowInputOption={setShowInputOption}
-                      inputValue={inputValue}
-                      setInputValue={setInputValue}
+                      showInputOption={showIterationsInputOption}
+                      setShowInputOption={setShowIterationsInputOption}
+                      inputValue={iterationsInputValue}
+                      setInputValue={setIterationsInputValue}
                       setMenuIsOpen={setIterationsMenuIsOpen}
                     />
                 }}
               />
           </div>
+          <div className={styles.form_item}>
+              <label className={styles.form_item_label}>
+                {PROTOCOL_QUERY_EN.FIELDS_LABEL.MESSAGE_SIZE} <span className={styles.required}>{PROTOCOL_QUERY_EN.FIELDS_LABEL.REQUIRED}</span>
+              </label>
+              <AttSelect
+                className={styles.select_form_item}
+                options={messageSizeOptions}
+                placeholder=''
+                value={messageSize as AttSelectOption[]}
+                onChange={onMessageSizeChanged}
+                isMulti
+                hideSelectedOptions={false}
```

Comment:
I think the default value if false try to remove it and check if it working

Reply:
The default value is true. That's why we need it.